lib/advanced-open-file-view.coffee

```diff
@@ -20,6 +20,19 @@ isRoot = (inputPath) ->
   return path.dirname(inputPath) is inputPath
 
 
+absolutify = (inputPath) ->
+  ###
+  Ensure that the given path is absolute. Relative paths are treated as
+  relative to the current project root.
+  ###
+  if getRoot(inputPath) == "."
+    projectPaths = atom.project.getPaths()
+    if projectPaths.length > 0
+      return path.join(projectPaths[0], inputPath)
```

Comment:
I like the idea of simultaneously using all the project roots. I tried implementing this but it needs better UI to convey to the user just what it's doing. For now, I think I'll stick with the first project's path, and spend some time thinking about what the UI for multiple project support should look like.